### DISCUSSION TITLE
How can i put docusaurus into a subdirectory like /docs?

### DISCUSSION THREAD

**Discussion by bestony** [facebook/docusaurus#6653] [Q&A] [answered]
Hi, i build a npm package, and i want to use docusaurus as document site.
but i want to keep document with source code, and put docusaurus into `/docs` directory

**Comment by slorber** [comment, marked as answer]
🤷‍♂️ just create the dir and move docusaurus inside it 😅

**Reply by slorber** [reply]
Jest puts code + docs in same repo, Docusaurus is inside ./website

https://github.com/facebook/jest

**Comment by cangSDARM** [comment]
This seems a bit similar to mine: https://github.com/facebook/docusaurus/discussions/6689, I only have two questions to be solved, if solved I can answer yours
